vnext/Universal.UnitTests/Tests/CreateModulesTests.cpp

```diff
@@ -19,6 +18,8 @@
 #include <ViewManager.h>
 #include <cxxreact/MessageQueueThread.h>
 
+#include <CreateModules.h>
```

Comment:
OOI, what's the significance of this re-ordering?

Reply:
Oh, is that another form of auto-formatting?

---
In reply to: [320518170](https://github.com/microsoft/react-native-windows/pull/3057#discussion_r320518170) [](ancestors = 320518170)

Reply:
The clang formatting sorts groups of includes alphabetically. I need the CreateModules.h to be last in order for x86 to build properly, so I had to move it down into a new group.